lib/streamlit/elements/progress.py

```diff
@@ -29,6 +30,27 @@
 FloatOrInt: TypeAlias = Union[int, float]
 
 
+def _check_float_between(value: float, low: float = 0.0, high: float = 1.0) -> bool:
+    """
+    Checks given value is 'between' the bounds of [low, high],
+    considering close values around bounds are acceptable input
```

Comment:
I think it would be great if you extended the comment slightly explaining *why* we want to consider close values around bounds, e.g. `... acceptable input, for example in order to account for values being slightly off due to floating point operations.`. What do you think?

Reply:
I agree. Thank you for your comment. I have extended the docstring in commit [98ff93b4b3af986ef4fde89b8689a22c81dc7a09](https://github.com/streamlit/streamlit/pull/7953/commits/98ff93b4b3af986ef4fde89b8689a22c81dc7a09).